lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -230,7 +229,7 @@ export default function LMSFilePicker({
     case 'reload':
       continueButton = (
         <Button
-          disabled={dialogState.state === 'fetching'}
+          disabled={dialogState.state === 'reloading'}
```

Comment:
I created a new ticket for this issue. We can solve it later

Reply:
@seanh suggested to disabled this button while fetching the files. I think it is a good idea because, in contrast to the other buttons, this has undesirable consequences if you keep clicking.